paasta_tools/cli/cmds/spark_run.py

```diff
@@ -606,6 +636,30 @@ def get_spark_app_name(original_docker_cmd: Union[Any, str, List[str]]) -> str:
     return spark_app_name
 
 
+def _calculate_docker_memory_cpu_limit(spark_conf):
```

Comment:
nit: could we add types to any new functions we add? imo, it'd also be nice to separate this into two functions so that you don't need to look at the code to know what order memory/cpu are in

Reply:
Added